docker-compose.yml.example

```diff
@@ -35,7 +35,7 @@ services:
       GITHUB_CLIENT_SECRET: "<GitHub OAuth Client Secret>"
       GITHUB_SECRET_TOKEN: "<GitHub Secret Token>"
       SECRET_KEY_BASE: "<random 65 chars string>"
-      GITHUB_INTEGRATION_ID: "<GitHub Integration ID>"
+      GITHUB_APP_ID: "<GitHub Integration ID>"
```

Comment:
The description should also say `<GitHub App ID>`

Reply:
Oh, I see. Let me change